tests/gui/models/store.spec.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
Rather than create a new file, should these tests have been added to `tests/gui/models/flash-state.spec.js` ?